src/listeners/addFavorite.js

```diff
@@ -0,0 +1,14 @@
+import { Favorite } from "../classes/Favorite.js";
+/**
+ * 
+ * @param {import("@lenra/app-server").props} props 
+ * @param {import("@lenra/app-server").event} _event 
+ * @param {import("@lenra/app-server").Api} api 
+ */
+export async function addFavorite(props, _event, api) {
+    api.data.createDoc(new Favorite(props.session, "@me"));
```

Comment:
Tu n'as pas attendu le résultat de ton appel

```suggestion
    const favorite = await api.data.createDoc(new Favorite(props.session, "@me"));
```